app/src/main/java/me/ivg2/foodapp/Model/Recipe.java

```diff
@@ -43,11 +44,23 @@ public Recipe(String name, String imageUrl, String source, ArrayList<String> ing
         cookTimeMinutes = 0;
     }
 
+    private String formatTime(int minutes, int hours) {
+        String formattedCookTime = "";
+        if (hours > 0 && hours!=0) {
+            formattedCookTime += hours + "h ";
+        }
+
+        if (minutes > 0) {
+            formattedCookTime += minutes + "m ";
+        }
+        return formattedCookTime;
+    }
+
     public Recipe(String name, String source, int cookTimeHours, int cookTimeMinutes) {
         this.name = name;
         this.source = source;
-        this.cookTimeHours = cookTimeHours;
```

Comment:
You should still keep the cookTimeHours and cookTimeMinutes variables so that you can update them on edit.  Also, on setCookTimeHours() and setCookTimeMinutes() make sure to rerun formatTime() so that the formatted time is in sync with updates.